app/src/main/java/com/example/weather/domain/Repository.java

```diff
@@ -28,8 +30,7 @@
     Completable updateWeather(WeatherData weatherData);
 
     Observable<Pair<Float,String>> getCurrentWeatherDataByCoord(String lat, String lon, String units);
-
-    Observable<HashMap<String,String>> getDailyWeatherDataByCoord(String lat, String lon,int day , String units);
+    Observable<Pair<Float,String>> getDailyWeatherDataByCoord(String lat, String lon, int day , String units);
```

Comment:
You pass `units` to every request like argument. Could we set it once for every request using `Interceptor` for example?